src/query/api/v1/handler/graphite/find_test.go

```diff
@@ -241,4 +251,28 @@ func TestFind(t *testing.T) {
 	}
 
 	require.Equal(t, expected, r)
+	header := w.Header().Get(handler.LimitHeader)
+	if hasHeader {
+		require.Equal(t, "true", header)
+	} else {
+		require.Equal(t, "", header)
+	}
+}
+
+var limitTests = []struct {
+	name               string
+	ex, ex2, hasHeader bool
```

Comment:
Super nit: Maybe make a struct so passing to `testFind(...)` method is easier?

i.e.

```go
type findTestParams struct {
  terminatedQueryExhaustive bool
  childQueryExhaustive bool
  expectHeader bool
}

var limitTests = []struct {
  name string
  params findTestParams
}{
  {"both incomplete", findTestParams{false, true, true}},
  // ...
}

func testFind(t *testing.T, p findTestParams) {
  // ...
}
```

And similarly for `setupStorage(...)` could we use a `type setupStorageOptions struct` so that as we add new fields we don't need to update each test calling into `setupStorage(t, opts)` individually by adding the new default params to the call to the method?

Reply:
I don't mind as much for tests, seems like it just makes it slightly more complex to me, but changed this in prod code